src/Sulu/Bundle/MediaBundle/Tests/Unit/Search/Subscriber/MediaSearchSubscriberTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Tests\Unit\Search\Subscriber;
+
+use Sulu\Bundle\MediaBundle\Entity\Media;
+use Sulu\Bundle\MediaBundle\Entity\File;
+use Sulu\Bundle\MediaBundle\Media\Manager\MediaManagerInterface;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\FileVersion;
+use Massive\Bundle\SearchBundle\Search\Event\PreIndexEvent;
+use Prophecy\PhpUnit\ProphecyTestCase;
+use Sulu\Bundle\MediaBundle\Search\Subscriber\MediaSearchSubscriber;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Sulu\Bundle\SearchBundle\Search\Document;
+use Prophecy\Argument;
+use Massive\Bundle\SearchBundle\Search\Metadata\IndexMetadata;
+
+class MediaSearchSubscriberTest extends ProphecyTestCase
+{
+    private $mediaManager;
+    private $subscriber;
+    private $metadata;
+    private $indexMetadata;
+    private $fileVersionMeta;
+    private $fileVersion;
+    private $file;
+    private $media;
+    private $event;
+    private $document;
+    private $reflection;
```

Comment:
IMHO it's really nice to get the available methods if you add the mocked type (e.g. `MediaManagerInterface`).
But not that important to me.